yarn.lock

```diff
@@ -4156,6 +4156,13 @@
     fireblocks-sdk "^3.1.4"
     web3-providers-http "1.8.0"
 
+"@fordefi/web3-provider@^0.2.0":
+  version "0.2.0"
+  resolved "https://registry.yarnpkg.com/@fordefi/web3-provider/-/web3-provider-0.2.0.tgz#c48da777410081e29e6ccdae6e40c8dc1558f395"
+  integrity sha512-7zBzg/F8fJgspmjeKbI3QjNKLYmycCtWOuPf225Ye0O+8zRtQr6PgaCh2tDhmFMa3gmAJ+BAhuPqCyYyNw4Y+w==
+  dependencies:
+    viem "^2.8.6"
```

Comment:
oh this might cause some issues since we need to pin our SDK to `2.8.6` having `^2.8.6` as a dependency will have yarn (as you can see below it maps `^2.8.6` to `2.9.32`) pull in some other version.

Reply:
removed